server/src/main/java/org/elasticsearch/index/query/RangeQueryBuilder.java

```diff
@@ -306,10 +306,25 @@ public RangeQueryBuilder relation(String relation) {
     protected void doXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject(NAME);
         builder.startObject(fieldName);
-        builder.field(FROM_FIELD.getPreferredName(), maybeConvertToString(this.from));
-        builder.field(TO_FIELD.getPreferredName(), maybeConvertToString(this.to));
-        builder.field(INCLUDE_LOWER_FIELD.getPreferredName(), includeLower);
-        builder.field(INCLUDE_UPPER_FIELD.getPreferredName(), includeUpper);
+
+        Object from = maybeConvertToString(this.from);
+        if (from != null) {
```

Comment:
Just double checking that we don't want to continue to output `null` values for gt/get etc... for bwc reasons. I haven't really thought through why we would want or need to, just want to double check your opinion and reasioning on this.

Reply:
I'm thinking of this as a totally new output, where we can decide what makes the most sense instead of trying to preserve similar behavior. To me, it doesn't really help to output something like `"gt": null` when no lower bound has been set. I don't think this is how users think about a missing lower bound (that the value must be greater than null)?

 To check I'm not missing something, do you have a specific concern around BWC and null values?

Reply:
No, no particular concern, just asking.